SpringParking-parkingBranch/src/main/java/pl/springDemo/demo/user/User.java

```diff
@@ -0,0 +1,54 @@
+package pl.springDemo.demo.user;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import pl.springDemo.demo.userPayment.UserPayment;
+
+import javax.persistence.*;
+import java.util.List;
+import java.util.Objects;
+
+@Entity
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
```

Comment:
dlaczego generujesz **publiczny** konstruktor bezparametrowy?

Reply:
Zgodnie ze specyfikacją encja powinna mieć konstruktor bezargumentowy public albo protected, a w ten sposób mogłam zaoszczędzić kolejne linijki kodu i zachować czystość kodu. Oczywiście mogłam dodać do adnotacji fragment (acces = AccessLevel.PROTECTED) jednak wciąż jest to zgodne z dokumentacją